### PR TITLE
Point MAINTAINERS to the one in the main Emissary repo

### PR DESCRIPTION
**Pull request by kflynn**
Rather than trying to keep both files up to date, point this one at
the other one. (I don't want to point the other one here because I
expect most people contributing to look at the Emissary repo rather
than the community repo.)

Signed-off-by: Flynn <emissary@flynn.kodachi.com>

### CODE REVIEW

**Review by AliceProxy — APPROVED** [review]

**Comment by kflynn** [comment]
I'm personally inclined to close this repo now that `emissary.git` is online, but figured one step at a time – there're more folks needed for that conversation!

**Review by LanceEa — APPROVED** [review]
lgtm

**Comment by LanceEa** [comment]
@kflynn - sounds good let's chat in the emissary-maintainers channel and for now there is no reason to hold this PR up so feel free to merge when ready.

**Pull request merged**
